stable/coredns/README.md

```diff
@@ -0,0 +1,84 @@
+CoreDNS
+=======
+
+CoreDNS is a DNS server that chains middleware and provides Kubernetes DNS Services
+
+TL;DR;
+------
+
+```console
+$ helm install --name coredns stable/coredns
+```
+
+Introduction
+------------
+
+This chart bootstraps a [CoreDNS](https://github.com/coredns/coredns) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+This chart will provide DNS Services to a Kubernetes cluster and is a drop-in replacement for Kube/SkyDNS.
+
+Prerequisites
+-------------
+
+-	Kubernetes 1.4+ with Beta APIs enabled
+
+Installing the Chart
+--------------------
+
+Since there is typically only a single DNS Server per kubernetes cluster, this Chart does not make use of the release name. For convenience the chart can be installed as follows:
+
+```console
+$ helm install --name coredns stable/coredns
+```
```

Comment:
You may want to mention that the RC or Deployment for Kube-DNS can be deleted after this is deployed.